config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -16,6 +16,17 @@ function cleanup()
 
 trap cleanup EXIT
 
+case ${BOARD} in
+
+    octopus)
+        SERIAL="ttyS1"
+        ;;
+
+    *)
+        SERIAL="ttyS0"
+        ;;
+esac
+
```

Comment:
How about adding a parameter to `chromiumos-octopus` in `rootfs-configs.yaml` instead?

Reply:
Updated according suggestion

Reply:
Ah I thought there was some generic `params` attribute for rootfs config objects but it looks like there isn't. Maybe that feature should be moved to the base YAML config object. But that's fine, it's some improvement for later. The approach of adding an attribute explicitly in the config object works as well.